{"name": "Вход курьера с правильным логином, но не правильным паролем с возвращением ошибки-", "status": "passed", "start": 1725392776350, "stop": 1725392776717, "uuid": "689a02e2-2f4e-4af7-a17b-3c39010663c8", "historyId": "5c33c763c7fa0a0e868e1251ebb5bd07", "testCaseId": "5c33c763c7fa0a0e868e1251ebb5bd07", "fullName": "tests.test_login_courier.TestLoginCourier#test_correct_login_with_incorrect_password", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login_courier"}, {"name": "subSuite", "value": "TestLoginCourier"}, {"name": "host", "value": "DESKTOP-ECR6Q27"}, {"name": "thread", "value": "3304-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login_courier"}]}